### DISCUSSION TITLE
How do I save the whole image when I only Inpainted a small portion?

### DISCUSSION THREAD

**Discussion by vbmark** [zero01101/openOutpaint#264] [Q&A]
I select a small area to inpaint and do a batch.  I want to keep 2 of the 5 and then do another batch.

However, it seems that my only options are to save the "Dream" box area or click "Y" to apply and save only that one and loose the other(s).

Is there a way to save the entire image while not closing the inpaint area?

Thank you.

**Comment by zero01101** [comment]
sorry for the delay; just to make sure i'm following, are you wanting to save some or all of the other options presented in the full image? if that's the case, it's not "supported" per se- the batch count/size gives you options to select from, but you do only get to make a single final choice.  you could potentially use the marquee select tool to copy the image before inpainting to a stamp and use the same seed to inpaint with, stamping your original image on several layers and then just disabling the visibility of each layer as desired, and export/save each image that way... it's not exactly convenient, but unfortunately your use case as i'm understanding is not one openOutpaint was built with in mind :/

**Comment by vbmark** [comment]
Hello, 

I'll play around with the layering suggestion.  

However, I hope that some day there will be an option to save the entire, visible image and just ignore the inpaint box.

I appreciate you replying.
